### PR TITLE
add user_auth

### PR DESCRIPTION
**Pull request by Spidey03**

### CODE REVIEW

File: user_auth/admin.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# your django admin
+from django.contrib import admin
+from user_auth.models import User
+from django.contrib.auth.admin import UserAdmin
+
+
+admin.site.register(User, UserAdmin)
```
Comment (raviteja1766) [inline user_auth/admin.py]: Add new line at the end of the file

File: user_auth/presenters/presenter_implementation.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+
+from common.dtos import UserAuthTokensDTO
+from covid_dashboard.constants.exception_messages \
+    import INVALID_USERNAME, INVALID_PASSWORD, USER_NOT_ADMIN
+from covid_dashboard.interactors.presenters.presenter_interface\
+    import PresenterInterface
+from django_swagger_utils.drf_server.exceptions import NotFound, Forbidden
```
Comment (raviteja1766) [inline user_auth/presenters/presenter_implementation.py]: External imports at the top of the file #CleanCode

File: user_auth/views/user_login/__init__.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+API_ENVIRONMENT = "ENV_MOCK"  # ENV_MOCK / ENV_IMPL
```
Comment (raviteja1766) [inline user_auth/views/user_login/__init__.py]: Change into implementation from mock

File: ib_miniprojects_backend/settings/.~c9_invoke_7WVLii.py [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+
+STAGE = os.environ.get("STAGE", "local")
+
+AUTH_USER_MODEL = 'user_auth.User'
+
+DEFAULT_OAUTH_APPLICATION_NAME = "ib_miniprojects_backend"
+DEFAULT_OAUTH_CLIENT_ID = ""
+DEFAULT_OAUTH_CLIENT_SECRET = ""
+DEFAULT_OAUTH_SCOPES = "read write"
+DEFAULT_ACCESS_TOKEN_EXPIRY_IN_SECONDS = 10000000
```
Comment (raviteja1766) [inline ib_miniprojects_backend/settings/.~c9_invoke_7WVLii.py]: add new online

File: user_auth/constants/exception_messages.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    "district not found", "INVALID_DISTRICT_ID"
+)
+
+INVALID_MANDAL_ID = (
+    "mandal not found", "INVALID_MANDAL_ID"
+)
+
+USER_NOT_ADMIN = (
+    "you can access this", "USER_NOT_ADMIN"
+)
```
Comment (raviteja1766) [inline user_auth/constants/exception_messages.py]: add new line

**Review by raviteja1766 — CHANGES_REQUESTED** [review]
Change those updates

File: ib_miniprojects_backend/settings/.~c9_invoke_7WVLii.py (hunk 4 above)
Comment (Spidey03) [reply ib_miniprojects_backend/settings/.~c9_invoke_7WVLii.py]: I have deleted the file....
It is an unnecessary file

**Pull request merged**